SampleGoGrpc/SimpleMath/grpc.go

```diff
@@ -0,0 +1,259 @@
+//Generated by GRPC Go plugin
+package SimpleMath
+
+import "github.com/google/flatbuffers/go"
+
+import (
+  grpc "google.golang.org/grpc"
+  context "golang.org/x/net/context"
+)
+
+// FlatCodec implements Grpc-go Codec
+
+type FlatCodec struct{}
+
+func (FlatCodec) Marshal(v interface{}) ([]byte, error) {
+  return v.(*flatbuffers.Builder).FinishedBytes(), nil
+}
+
+func (FlatCodec) Unmarshal(data []byte, v interface{}) error {
+  v.(flatbuffersInit).Init(data, flatbuffers.GetUOffsetT(data))
+  return nil
+}
+
+func (FlatCodec) String() string {
+  return "flatbuffers"}
+
+type flatbuffersInit interface {
+  Init(data []byte, i flatbuffers.UOffsetT)
+}
+
+// Client API for Math service
+type MathClient interface{
+  Mod2(ctx context.Context, in *flatbuffers.Builder, opts... grpc.CallOption) (* NumRes, error)  
+  AddNumbers(ctx context.Context, opts... grpc.CallOption) (Math_AddNumbersClient, error)  
+  Squares(ctx context.Context, in *flatbuffers.Builder, opts... grpc.CallOption) (Math_SquaresClient, error)  
+  IsOdd(ctx context.Context, opts... grpc.CallOption) (Math_IsOddClient, error)  
+}
+
+type mathClient struct{
+  cc *grpc.ClientConn
+}
+
+func NewMathClient(cc *grpc.ClientConn) MathClient {
+  return &mathClient{cc}
+}
+
+func (c *mathClient) Mod2(ctx context.Context, in *flatbuffers.Builder, opts... grpc.CallOption) (* NumRes, error){
+  out := new(NumRes)
+  err := grpc.Invoke(ctx, "/SimpleMath.Math/Mod2", in, out, c.cc, opts...)
+  if err != nil { return nil, err }
+  return out, nil
+}
+
+func (c *mathClient) AddNumbers(ctx context.Context, opts... grpc.CallOption) (Math_AddNumbersClient, error){
+  stream, err := grpc.NewClientStream(ctx, &_Math_serviceDesc.Streams[0], c.cc, "/SimpleMath.Math/AddNumbers", opts...)
+  if err != nil { return nil, err }
+  x := &mathAddNumbersClient{stream}
```

Comment:
How can it be avoided?
It was exactly similar in the [protobuf grpc-go generator](https://github.com/grpc/grpc-go/blob/master/examples/helloworld/helloworld/helloworld.pb.go),